### ISSUE TITLE
[Feature]: Better visual separation between icons in the updated toolbar (1.13.0b1)

### ISSUE THREAD

**Issue by melek** [RPTools/maptool#3893]
### Feature Request

There are a few UI adjustments that might tighten up the toolbar. 

### The Solution you'd like

1. **Consistent separators**. The line separator between the main tool selection and the sub-tool selection feels like it ought to be centered / have equal padding on either side, closer to the separator to the left of the 'Select Map' button. Maybe all the separators could use some adjustment to be consistent; this is the worst one, the one by the volume control is a bit far from the view controls but OK, and the one by 'Select Map' is perhaps just right. Here is the separator I mean, a sliver next to the VBL Tools button:  
![image](https://user-images.githubusercontent.com/220149/229292530-ba5fe6c9-fb3e-42f2-97ff-aed1c37040ae.png)
1. **Clearer separation between Blocking Layer selection and BL tools**. The new non-alpha blocking layer selection is a bit hard to parse visually between sub-tools and layer selection:  
![image](https://user-images.githubusercontent.com/220149/229291843-cc3e8dab-50b9-4910-bc4b-5b219722a1a1.png)  
In contrast, the old icons used letters which distinguished them a little more clearly:  
![image](https://user-images.githubusercontent.com/220149/229291992-bf550e73-8cc5-47c2-bed7-bd8cb9a9713e.png)  
Blocking layer tools are pretty enigmatic to a new user either way, but even as an experienced user I think a separator may help.
1. **Slightly more right-side margin or padding on the Select Map button**. It just feels a bit too close to the right-side of the toolbar. Either some spacing in or to the right of the button to balance the toolbar, perhaps the width of the handle on the left, would feel nice. Here is how it looks now undocked to accentuate the button's closeness to the edge:  
![image](https://user-images.githubusercontent.com/220149/229292579-ad5fa2c2-68d6-4f95-a371-73447a432d28.png)

Perhaps others have additional suggestions to tighten up the toolbar now that we have these new icons, but these are the ones that struck me first. 

### Alternatives that you've considered.

Dealing with it.

### Additional Context

Windows 10

**Comment by kwvanderlinde** [comment]
@melek Can you check whether the tweaks made fulfill the request?

**Comment by kwvanderlinde** [comment]
Closing this off as it's a minor tweak that I'm pretty confident is addressed. If that's wrong we can reopen it again.

![image](https://github.com/RPTools/maptool/assets/7492219/4979ffe6-0bef-4309-8fb1-f5bbfb0546ce)

![image](https://github.com/RPTools/maptool/assets/7492219/330df6e1-6a7a-499e-9572-29805c8be6ea)
